xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -770,9 +770,9 @@ This method is a no-op and added only for consistency: the `value` is returned u
       </Parameters>
       <Docs>
         <param name="source">The read-only span of bytes to read.</param>
-        <param name="value">When the method returns, the value read out of the read-only span of bytes, as little endian.</param>
+        <param name="value">When this method returns, the value read out of the read-only span of bytes, as little endian.</param>
         <summary>Reads an <see cref="T:System.Int16" /> out of a read-only span of bytes as little endian.</summary>
-        <returns>If the span is too small to contain an <see cref="T:System.Int16" />, return <see langword="false" />; otherwise, return <see langword="true" />.</returns>
+        <returns><see langword="true" /> if the span can contain an <see cref="T:System.Int16" />; otherwise, <see langword="false" />.</returns>
```

Comment:
@mairaw, I'll defer to you on this, but I am curious to know why you removed the too small/too large qualifier. I thought specifying what it means when a "span can contain an int" explicitly would be helpful.

Reply:
@mairaw I reverted this particular change to the original wording, but let me know if you'd like us to keep yours instead.

Reply:
I made the change before because we always state the true condition first but if you guys don't like it, I won't block this from merging because of it.

Reply:
Its fine to keep the "true" condition first. My question here was specifically around the text for the true condition.

**Before:**
`true if the span can contain...`
**Now:**
`true if the span is large enough to contain...`

We still have the true condition stated first.

Reply:
sure, that's fine